proto/rill/runtime/v1/resources.proto

```diff
@@ -448,6 +448,7 @@ message ExplorePreset {
   optional string timezone = 12;
   optional string time_grain = 13;
   optional string select_time_range = 14;
+  optional string time_column = 33;
```

Comment:
nit: call this `time_dimension` to reflect the terminology we use elsewhere ("dimension" basically meaning something the metrics layer can aggregate by, which may not be an actual column in a database table).